lib/fixTriple.js

```diff
@@ -56,16 +56,20 @@ export default function fixTriple(triple) {
  * @param term the term to be fixed
  */
 function fixTerm(term) {
-  if(validateTerm(term)) {
+  if (validateTerm(term)) {
     return term;
   } else {
-    if(term.datatype) {
-      if(term.datatype.value === 'http://www.w3.org/2001/XMLSchema#boolean') {
+    if (term.datatype) {
+      if (term.datatype.value === 'http://www.w3.org/2001/XMLSchema#boolean') {
         return fixBoolean(term);
-      } else if(term.datatype.value === 'http://www.w3.org/2001/XMLSchema#date') {
+      } else if (term.datatype.value === 'http://www.w3.org/2001/XMLSchema#date') {
         return fixDate(term);
-      } else if(term.datatype.value === 'http://www.w3.org/2001/XMLSchema#dateTime') {
+      } else if (term.datatype.value === 'http://www.w3.org/2001/XMLSchema#dateTime') {
         return fixDateTime(term);
+      } else if (term.datatype.value === 'http://www.w3.org/2000/01/rdf-schema#Literal') {
+        return fixLiteral(term);
+      } else if (term.datatype.value == 'http://www.w3.org/1999/02/22-rdf-syntax-ns#langString') {
+        return fixLiteral(term);
```

Comment:
**Question:** @nbittich Is it normal to use `fixLiteral` in language condition?

Reply:
@aliokan  yeah, in that case it meant that the language value was empty, so we don't have a language (related to `Is this new scrapper handles types issues we have with http://www.w3.org/1999/02/22-rdf-syntax-ns#langString without lang attribute ` ). If we don't have a language attribute, it should be a normal string  literal

Reply:
hmm but shouldn't we check there's no lang value?